control/server.py

```diff
@@ -58,6 +58,20 @@ def int_to_bitmask(n):
     """Converts an integer n to a bitmask string"""
     return f"0x{hex((1 << n) - 1)[2:].upper()}"
 
+def cpumask_set(args):
+    """Check if reactor cpu mask is set in command line args"""
+
+    # Check if "-m" or "--cpumask" is in the arguments
+    if "-m" in args or "--cpumask" in args:
+        return True
+
+    # Check for the presence of "-m=" or "--cpumask="
+    for arg in args:
+        if arg.startswith('-m=') or arg.startswith('--cpumask='):
+            return True
```

Comment:
This is wrong. You don't need "=" after "-m". Only in long parameters. And I don't think we need to even check for "=". this should be:

if arg.startswith('-m') or arg.startswith('--cpumask'):

we don't want to get into parsing issues like spaces before the "="